sat/cli/bootprep/input/image.py

```diff
@@ -639,14 +639,42 @@ def product_version(self):
         return self.image_data['base']['product'].get('version')
 
     @property
-    def base_description(self):
-        """str: a human-readable description of the base we are starting from"""
+    def filter_prefix(self):
+        """str or None: the filter prefix to use for filtering images or recipes from the product"""
+        return self.image_data['base']['product'].get('filter', {}).get('prefix')
```

Comment:
nit: could this be replaced with the following?
```suggestion
        return get_val_by_path(self.image_data,  'base.product.filter.prefix')
```

Reply:
Yeah, not sure how I forgot about that useful function.